pkg/cmd/domains/create/create.go

```diff
@@ -0,0 +1,90 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+
+	msg "github.com/aziontech/azion-cli/messages/domains"
+	api "github.com/aziontech/azion-cli/pkg/api/domains"
+
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+type Fields struct {
+	Name                 string   `json:"name"`
+	Cnames               []string `json:"cnames,omitempty"`
+	EdgeApplicationId    int      `json:"edge_application_id"`
```

Comment:
I think we are using int64 as a standard for the APIs

Reply:
by default int is int64

![image](https://user-images.githubusercontent.com/52722027/215193339-7f6e0c75-f387-4666-942e-0fedf937428b.png)